resources/compass/values.yaml

```diff
@@ -143,18 +134,24 @@ global:
+
   istio:
     gateway:
-      name: "kyma-gateway"
-      namespace: "kyma-system"
+      name: compass-istio-gateway
```

Comment:
By default we should have `kyma-gateway here`

Reply:
Will revert these values back.